shjq/README.md

```diff
@@ -0,0 +1,114 @@
+# shjq
+
+This is a generic JSON to smarthome plugin. Fetch any JSON encoded
+data via http(s) or from a file, extract the interesting data and feed
+the values to smarthome items.
+
+It uses the lightweight and flexible JSON processor jq [https://stedolan.github.io/jq/]
+
+# Dependencies
+
+This plugin requires
+- requests
+- requests-file
+- pyjq 
+
+# Configuration
+
+## Enable the plugin in etc/plugin.yaml
+
+You can add as many instances as you have JSON sources to process. Make sure to add a 
+unique instance name to each instance if you configure more than on instance. That
+instance name will serve as a key to the item configuration further down.
+
+### Plugin specific attributes
+#### url
+
+The address of the json data. Currently http://, https:// and file:// schemes are supported.
+Note: using absolute file paths for the file:// schema yields in tripple forward slashes. This
+might look funny but it is not an error.
+
+#### cycle
+
+The repetitive polling interval in seconds. Defaults to 30 seconds.
+
```

Comment:
I think we should put the documentation about the items into the `plugin.yaml` not directly in the `README.md` file. Examples and more details about how to use the plugin should be part of `README.md`.